Makefile

```diff
@@ -0,0 +1,9 @@
+render: sdf.md
```

Comment:
This needs all the included files as dependencies:
``` 
   1706:{::include sdf-framework.cddl}
   1721:{::include sdf.jso.json-unidiff}
```

Reply:
Should be fixed now via 8cd0cb778865d6dfc10b13ac4f963c7c3ff68745 :)